etc/tutorial/cylc-forecasting-suite/bin/forecast

```diff
@@ -187,7 +187,7 @@ def main(forecast_interval, forecast_itterations):
 
     # Get science settings.
     spline_level = int(os.environ.get('SPLINE_LEVEL', 0))
-    weighting = map(float, os.environ.get('WEIGHTING', '1').split(','))
+    weighting = list(map(float, os.environ.get('WEIGHTING', '1').split(',')))
```

Comment:
Another over-zelous `list()`?

Reply:
Map returns a generator whereas `weighting` is later accessed by index so must be a list.